mantidproject/mantid
# Minor REFL refactor and tweaks

To test:
- make sure all the REFL tests pass
- make sure there's documentation for the new background subtraction algorithm
